content/en/synthetics/private_locations/_index.md

```diff
@@ -531,32 +531,65 @@ You can then go to any of your API or Browser test creation form, and tick your
 
-The recommendation for disk size is to allocate ~ 10MiB/slot (1MiB/slot for API-only private locations).
+**Example:** For a private location running only Browser tests, with a maximum number of concurrent test runs of `10`, the recommendation for a safe usage is 
+~ 1.5Core CPU `(150mCores + (150mCores*10 test runs))`, ~ 10GiB memory `(150MiB + (1GiB*10 test runs))`, and ~ 100MiB disk `(10MiB*10 test runs)`.
+
+**Note:** Resources requirements may vary based on the application being tested (size/number of assets to be loaded, etc.).
+
+**Note:** When running both API/Multistep API tests and Browser tests on a single private location, the recommendation is to perform computation using Browser tests resource requirements.
+
+### Assign resources to your private location
+
+Once you know about the [**total** requirements for your private location](#private-location-total-hardware-requirements), you can decide how you want these resources to be distributed:
+
+* You can assign all resources to a single worker (ie. to a single container). In this case:
+  * Set the [`concurrency` parameter][15] to `maximum number of test runs that can be executed in parallel on your private location`.
+  * Assign your [total private location resource requirements](#private-location-total-hardware-requirements) to your unique worker.
+* You can distribute resources across several workers. You can indeed run several containers for one private location with one single configuration file in order to spread the load. In this case:
+  * Set the [`concurrency` parameter][15] to `maximum number of test runs that can be executed on your private location / number of workers associated with your private location`.
+  * Assign `total private location resource requirements / number of workers` resources to each private location worker.
+
+**Example:** For a private location running only Browser tests, with a maximum number of concurrent test runs of `10`, your private location requires ~ 1.5 core CPU, ~ 10GiB memory, and ~ 100MiB disk. If you want to distribute these resources across 2 workers, the [`concurrency` parameter][15] should be set to `5`, and each worker should be allocated ~ 750mCores CPU, ~ 5GiB memory, and ~ 50MiB disk.
+
+#### Queueing mechanism
+
+When there are several workers associated with a private location, each worker requests a number of tests to run that depends on its [`concurrency` parameter][15] and on the number of additional test runs that can be assigned to it.   
+** Example:** 10 tests are scheduled to run simultaneously on a private location that has 2 workers running. If worker 1 is currently running 2 tests, it can request 3 additional tests to run. If at this time, worker 2 is not running any tests, it will then request the 5 following tests. The remaining 2 tests can be requested by which ever worker has finished running its test first, ie which ever worker has available slots.
```

Comment:
```suggestion
** Example:** Ten tests are scheduled to run simultaneously on a private location that has two workers running. If worker 1 is running two tests, it can request three additional tests to run. If worker 2 is not running any tests, it can request the five following tests. The remaining two tests can be requested by which ever worker has finished running its test first (which ever worker has available slots).
```